src/frontend/packages/ui/DateContext.ts

```diff
@@ -0,0 +1,7 @@
+import { createContext } from 'react';
+
+type DateContext = {
+    since: number,
+    until: number,
+}
+export const DateContext = createContext({since:0, until:9999999999999});
```

Comment:
Same remark for initialization, and it's missing a newline at the end of file 
```suggestion
export const DateContext = createContext({since:0, until:9999999999999});

```

Reply:
Also default to last week now